File: Plugins/Aspose-Cells-Java-for-Jython/asposecells/WorkingWithWorksheets/RemovingWorksheetsusingSheetIndex.py
from asposecells import Settings
from com.aspose.cells import Workbook
from java.io import FileInputStream;


class RemovingWorksheetsusingSheetIndex:

    def __init__(self):
        dataDir = Settings.dataDir + 'WorkingWithWorksheets/RemovingWorksheetsusingSheetIndex/'
          
        fstream=FileInputStream(dataDir + "Book1.xls");

        #Instantiating a Workbook object with the stream
        workbook = Workbook(fstream)

        #Removing a worksheet using its sheet index
        workbook.getWorksheets().removeAt(0)

        #Saving the Excel file
        workbook.save(dataDir + "book.out.xls")

        #Closing the file stream to free all resources
        fstream.close()


        #Print Message
        print "Sheet removed successfully."

if __name__ == '__main__':        
    RemovingWorksheetsusingSheetIndex()

File: Examples/src/main/java/com/aspose/cells/examples/worksheets/RemovingWorksheetsusingSheetIndex.java
package com.aspose.cells.examples.worksheets;

import java.io.FileInputStream;

import com.aspose.cells.*;
import com.aspose.cells.examples.Utils;

public class RemovingWorksheetsusingSheetIndex {

	public static void main(String[] args) throws Exception {

		// The path to the documents directory.
		String dataDir = Utils.getSharedDataDir(RemovingWorksheetsusingSheetIndex.class) + "worksheets/";

		// Creating a file stream containing the Excel file to be opened
		FileInputStream fstream = new FileInputStream(dataDir + "book.xls");

		// Instantiating a Workbook object with the stream
		Workbook workbook = new Workbook(fstream);

		// Removing a worksheet using its sheet index
		workbook.getWorksheets().removeAt(0);

		// Saving the Excel file
		workbook.save(dataDir + "RWUsingSheetIndex_out.xls");

		// Closing the file stream to free all resources
		fstream.close();

		// Print Message
		System.out.println("Sheet removed successfully.");

	}
}
